benchmark_runner/common/virtctl/virtctl.py

```
@@ -37,10 +37,11 @@ def get_ssh_public_key(self, key_path: str) -> str:
             return f.read().strip()
 
     @typechecked
-    def virtctl_ssh(self, vm_name: str, command: str, namespace: str = '', key_path: str = '', username: str = '') -> Optional[str]:
+    def virtctl_ssh(self, vm_name: str, command: str, namespace: str = '', key_path: str = '', username: str = '', timeout: int = None) -> Optional[str]:
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)